cmd/kubeadm/app/apis/kubeadm/v1alpha1/defaults.go

```diff
@@ -42,6 +42,8 @@ const (
 	DefaultAuthorizationModes = "Node,RBAC"
 	// DefaultCertificatesDir defines default certificate directory
 	DefaultCertificatesDir = "/etc/kubernetes/pki"
+	// DefaultCertificatesBackupDir defines default certificate backup directory
+	DefaultCertificatesBackupDir = "/etc/kubernetes/pki/backup"
```

Comment:
I would not create this constant, instead just compute the `backup` subfolder or certDir, whatever it is